packages/apps-electron/webpack.renderer.config.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017-2020 @polkadot/apps authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+/* eslint-disable @typescript-eslint/camelcase */
+
+const merge = require('webpack-merge');
+const baseConfig = require('@polkadot/apps/webpack.base.config');
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const findPackages = require('../../scripts/findPackages');
+
+const ENV = process.env.NODE_ENV || 'development';
+const isProd = ENV === 'production';
+const context = __dirname;
+
+module.exports = merge(
+  baseConfig({
+    alias: findPackages().reduce((alias, { dir, name }) => {
```

Comment:
Would it be possible to pull up the `alias` to the `base` config as well? It's the same here and in the webapp.